crates/miden-lib/src/account/faucets/mod.rs

```diff
@@ -18,6 +19,30 @@ use crate::account::{auth::RpoFalcon512ProcedureAcl, components::basic_fungible_
 // BASIC FUNGIBLE FAUCET ACCOUNT COMPONENT
 // ================================================================================================
 
+// Initialize the digest of the `distribute` procedure of the Basic Fungible Faucet only once.
+static BASIC_FUNGIBLE_FAUCET_DISTRIBUTE: LazyLock<Word> = LazyLock::new(|| {
+    let distribute_proc_name = QualifiedProcedureName::new(
+        Default::default(),
+        ProcedureName::new(BasicFungibleFaucet::DISTRIBUTE_PROC_NAME)
+            .expect("failed to create name for 'distribute' procedure"),
+    );
+    basic_fungible_faucet_library()
+        .get_procedure_root_by_name(distribute_proc_name)
+        .expect("Basic Fungible Faucet should contain 'distribute' procedure")
+});
+
+// Initialize the digest of the `burn` procedure of the Basic Fungible Faucet only once.
+static BASIC_FUNGIBLE_FAUCET_BURN: LazyLock<Word> = LazyLock::new(|| {
+    let burn_proc_name = QualifiedProcedureName::new(
+        Default::default(),
+        ProcedureName::new(BasicFungibleFaucet::BURN_PROC_NAME)
+            .expect("failed to create name for 'burn' procedure"),
+    );
+    basic_fungible_faucet_library()
+        .get_procedure_root_by_name(burn_proc_name)
+        .expect("Basic Fungible Faucet should contain 'burn' procedure")
+});
+
```

Comment:
I think it'd be great to add tests for these to make sure that if they go out of sync with the MASM code, that we catch that fact in CI. Some of those are used naturally in tests, but `BasicFungibleFaucet::burn_digest` for example isn't used.

I think I'd just add a test for each component that calls all its procedure digest functions, just to make sure that if they panic, we notice that. Not sure we can implement something much better.